android/app/src/main/java/org/fossasia/openevent/utils/ConstantStrings.java

```diff
@@ -21,6 +21,8 @@
 
     public static final String APP_NAME = "APP_NAME";
 
+    public static final String IS_AUTH_ENABLED = "is_auth_enabled";
```

Comment:
In Application class it is `public static final String AUTH_OPTION = "is-auth-enabled";`.
I'm trying to follow naming convention.
JSON -> Kebab case
SharedPreference -> Snack case

Will it be okay?